pymobiledevice3/services/dvt/instruments/process_control.py

```diff
@@ -1,3 +1,5 @@
+import signal
```

Comment:
Unused import

Reply:
Sure

Reply:
still there

Reply:
answered from my phone to confirm. now it's fixed :)